Automattic/wp-calypso
# Framework: Import React addons individually

This should rid us of deprecation warnings.

To test:
* Verify that Calypso still works as expected.
* Verify that `Warning: require('react/addons') is deprecated. Access using require('react-addons-{addon}') instead` warnings are gone.
